docs/compass/03-01-security.md

```diff
@@ -217,3 +218,32 @@ foo@bar.com:
 **Scopes**
 
 Scopes are added to the authentication session in Tenant Mapping Handler. The handler gets not only `tenant`, but also `scopes`, which are fixed regarding of type of the object (Application / Runtime). Application and Runtime are always have the same scopes defined.
+
+### One-time token
+
+**Used by:** Runtime/Application
+
+**Connector flow:**
+
+1. Runtime/Application makes a call to the Connector's internal API.
+1. The OathKeeper uses the Token Resolver as a mutator. 
+The Client ID is extracted from the one-time token's `Connector-Token` header or from the `token` query parameter, and is then written to the `Client-Id-From-Token` header. In the case of failure, the header is empty.
+1. The OathKeeper proxies the request further to the Compass Gateway.
+1. The request is forwarded to the Connector.
+
+![Auth](./assets/token-security-diagram-connector.svg)
+
+**Director Flow:**
+
+1. Runtime/Application makes a call to the Director.
+1. The OathKeeper uses the Token Resolver as a mutator. The Client ID is extracted from the one-time token's `Connector-Token` header or from the `token` query parameter, and is then written to the `Client-Id-From-Token` header. In the case of failure, the header is empty.
+1. The call is then proxied to the Tenant mapping handler mutator, where the Client ID is mapped onto the `tenant` and returned to the OathKeeper. 
+1. Hydrator passes the response to ID_Token mutator which constructs a JWT token with scopes and tenant in the payload.
+1. The OathKeeper proxies the request further to the Compass Gateway.
+1. The request is forwarded to the Director.
+
+![Auth](./assets/token-security-diagram-director.svg)
+
+**Scopes**
+
+Scopes are added to the authentication session in Tenant Mapping Handler. The handler gets not only `tenant`, but also `scopes`, which are fixed regarding of type of the object (Application/Runtime). Application and Runtime always have the same scopes defined.
```

Comment:
```suggestion
Scopes are added to the authentication session in the Tenant Mapping Handler. The handler gets not only `tenant` but also `scopes` which are fixed regardless of the object type (Application/Runtime). Application and Runtime always have the same scopes defined.
```

Reply:
That's not what this paragraph is trying to say, but it should be fixed anyway. Thanks for pointing it out! I missed this error.